Vagrantfile

```diff
@@ -19,10 +22,12 @@ Vagrant.configure("2") do |config|
 
 		vmCfg.vm.network "private_network", ip: "172.27.10.5"
 
-		vmCfg.vm.provider "vmware_fusion" do |v|
-			v.vmx["memsize"] = "1024"
-			v.vmx["numvcpus"] = "2"
-			v.vmx["ethernet1.virtualDev"] = "vmxnet3"
+		["vwmare_fusion", "vmware_workstation"].each do |p|
```

Comment:
Thank you for making this work for users of Windows as this will be usability boon for developers using Windows.

Reply:
It doesn't quite work yet - I need to fix `vagrant-winnfs` too, but this is useful for larger dev boxes than Apple currently make.